lib/docker-compose/models/compose_container.rb

```diff
@@ -164,6 +166,14 @@ def prepare_environment(env_entries)
   end
 
   #
+  # Forces the labels structure to use the hash format.
+  #
+  def prepare_labels(labels)
```

Comment:
If a label isn't provided in YAML file, it would be nice to give the container a random label, because we will need at least one label when mapping the running containers with the next execution o Compose.